rpcd/playbooks/roles/rpc_maas/files/plugins/jbod_checker.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
jbod_checker.sh probably needs to be marked executable to keep it in line with the other plugins.